bot/cogs/information.py

```diff
@@ -32,20 +33,18 @@ def __init__(self, bot: Bot):
         # Sort the roles alphabetically and remove the @everyone role
         roles = sorted(ctx.guild.roles[1:], key=lambda role: role.name)
 
-        # Build a string
-        role_string = ""
+        # Build a list
+        role_list = []
         for role in roles:
-            role_string += f"`{role.id}` - {role.mention}\n"
+            role_list.append(f"`{role.id}` - {role.mention}")
 
         # Build an embed
         embed = Embed(
-            title="Role information",
-            colour=Colour.blurple(),
-            description=role_string
+            title=f"Role information (Total {len(roles)} roles)",
+            colour=Colour.blurple()
         )
-        embed.set_footer(text=f"Total roles: {len(roles)}")
 
-        await ctx.send(embed=embed)
+        await LinePaginator.paginate(role_list, ctx, embed)
```

Comment:
```suggestion
        await LinePaginator.paginate(role_list, ctx, embed, empty=False)
```
Setting the `empty` kwarg to `False` will remove the newline in between every role.

Before:
![image](https://user-images.githubusercontent.com/5323929/75840387-1613d780-5d99-11ea-9ea7-bf0fd8c4acd8.png)

After:
![image](https://user-images.githubusercontent.com/5323929/75840407-21ff9980-5d99-11ea-9b13-361da40ee218.png)